postgres-snapshots/manifests/pg-consumer.yaml

```diff
@@ -31,29 +31,6 @@ spec:
                       --dbname ${DB_NAME} \
                       --command 'SELECT NOW()'
               done
-          env:
-            - name: DB_HOST
-              valueFrom:
-                secretKeyRef:
-                  name: postgres-secret
-                  key: DB_HOST
-            - name: DB_PORT
-              valueFrom:
-                secretKeyRef:
-                  name: postgres-secret
-                  key: DB_PORT
-            - name: DB_USER
-              valueFrom:
-                secretKeyRef:
-                  name: postgres-secret
-                  key: DB_USER
-            - name: DB_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: postgres-secret
-                  key: DB_PASSWORD
-            - name: DB_NAME
-              valueFrom:
-                secretKeyRef:
-                  name: postgres-secret
-                  key: DB_NAME
+          envFrom:
+            - secretRef:
+                name: postgres
```

Comment:
updated this to match the concise version from the guide

Reply:
Actually, I thought about this and felt that it's good to have both versions. This one makes it more explicit what's in the secret, which is why I left it as is and used secretRef in the pg-snapshot pod. It's a minor thing though

Reply:
I think if you're referencing the values of a secret, you might as well also reference the keys, but I see your argument -- having it the longer way communicates the structure of the vars without having to switch files. Is it okay if I leave this as is for this example?